staging/src/k8s.io/cri-api/pkg/apis/services.go

```diff
@@ -133,4 +133,6 @@ type ImageManagerService interface {
 	RemoveImage(ctx context.Context, image *runtimeapi.ImageSpec) error
 	// ImageFsInfo returns information of the filesystem that is used to store images.
 	ImageFsInfo(ctx context.Context) ([]*runtimeapi.FilesystemUsage, error)
+	// ImageFsInfoV2 returns information of the image and continaer fileystems
+	ImageFsInfoV2(ctx context.Context) (*runtimeapi.ImageFsInfoResponse, error)
```

Comment:
Note: this approach was taken to allow ImageFsInfo to be stable (as in: not change the function signature. Also note: this is *not* the CRI API that CRIs will fulfill. This is a compatibility layer used by [kubelet](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kuberuntime/instrumented_services.go#L31C29-L31C29) and [cri-tools](https://github.com/kubernetes-sigs/cri-tools/blob/f8733ddf69f86dae2d40e9f690785ef4bdb4440e/pkg/framework/framework.go#L35) 
as such, it's not clear we provide any stability guarantees for the ImageManagerService interface: it all looks to be APIs used internally within projects SIG node owns. If we don't provide any stability guarantees for these APIs, then changing the existing ImageFsInfo endpoint will be cleaner and easier to maintain long term. If we do, we'll need this v2 approach.

Reply:
cc @dchen1107 @SergeyKanzhelev @mrunalp

Reply:
I think we should change the signature. We should be able to update the set of projects that use it.

Reply:
updated this. PTAL.

Reply:
fyi.. containerd has a copy of services.go for integration testing  https://github.com/containerd/containerd/blob/main/integration/cri-api/pkg/apis/services.go only used for integration testing back when we were supporting both cri v1alpha2 and v1 at the same time.. ( no longer needed )

Reply:
Thanks! So should we update it also? Or do you think that code can be removed?